src/json/indent.c

```diff
@@ -13,37 +15,47 @@ static void doIndent(int idt, char **o, const char *tab) {
 SDB_API char *sdb_json_indent(const char *s, const char *tab) {
 	int indent = 0;
 	int instr = 0;
-	int osz;
-	char *o, *O, *OE, *tmp;
+	size_t o_size = 0;
+	char *o, *O;
 	if (!s) {
 		return NULL;
 	}
-	osz = (1 + strlen (s)) * 20;
-	if (osz < 1) {
-		return NULL;
+
+	size_t tab_len = strlen (tab);
+	for (int idx = 0; s[idx]; idx++) {
```

Comment:
dont define `int` inside (). it is a c++ism and will not compile on many C compilers